modules/integration/tests-ui/src/test/resources/testng-server-mgt.xml

```diff
@@ -24,6 +24,7 @@
             <!--<class name="org.wso2.carbon.am.integration.ui.APIMANAGER2957EpochTimeTestCase"/>
             <class name="org.wso2.carbon.am.integration.ui.APIMANAGER3017ExternalStoreLifeCycleTestCase"/>
             <class name="org.wso2.carbon.am.integration.ui.APIMANAGER3149SSOLogin"/>-->
+            <!--<class name="org.wso2.carbon.am.integration.ui.APIResourceBodyParameterRemovalTestCase"/>-->
```

Comment:
Since these are selanium test cases and selanium is not properly configured in the build machine these test cases failed. Need to configure the builder machine and enable these test cases.